INSTALL.md

```diff
@@ -49,6 +49,42 @@ make test
 If you want to get rid of this environment, just run `make clean`.
 
 
-#### Development Environment Setup 
+## Development Environment Setup on Mac OS X
 
-//TODO: Coming soon...
\ No newline at end of file
+### Dependencies
+
+First you should install following things:
+
+XCode: Install it via the Apple Store application or in [Apple's developer site](https://developer.apple.com/xcode/download/)
+
+```shell
+brew install imagemagick@6 qt@5.5 homebrew/versions/mysql56 gs pkg-config
+brew install sphinx --with-mysql
+brew link qt@5.5 --force
+brew link imagemagick@6 --force
+```
+
+Also you'll need http://xquartz.macosforge.org/landing/ to run cucumber tests.
+
```

Comment:
I also needed `gem install spring -v 2.0.0`

Reply:
Well this is not necessary only if you want to use spring

Reply:
Then maybe we can add a section of `tips` or something like that for this kind of 'optional but helpful tools'

Reply:
Sure 👍 waiting for your commits ;)

Reply:
https://github.com/3scale/porta/pull/36 @hallelujah pointing to this branch 😄